babel.config.js

```diff
@@ -1,5 +0,0 @@
-module.exports = {
```

Comment:
How come you deleted this file?

Reply:
That might be an awkward decision I made to remove this file. If I keep this file, then it would lead to a problem/error that the authors of vue said they would optimize in next version, which is Vue 4.X. However, to coordinate different frameworks, i.e. axios, vue, expresss, whatever, the template mainly use Vue 3.X. And if we want to convert code from Vue 3.X to 4.X, it seems like there are many differences. Therefore, for this project, keep the same version with the template might be better. At least, some people have figured out the versions could coordinate together.

Reply:
Ok, deleting it is fine with me then.